recipe/meta.yaml

```diff
@@ -11,18 +11,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  skip: true  # [py<38]
```

Comment:
I don't see anything in `setup.py` that suggests that older python versions are not supported.

Reply:
Correct, you can drop the selector. The python versions we support are determined by the CBC file in aggregate

Reply:
ah indeed, right, at the very end, it will anyway be 38->3.11 ok perfect, thank you @marcoesters for the explanation

Reply:
```suggestion
  skip: true
```

Reply:
done

Reply:
If you remove this, you need to add `mock==2.0.0;python_version<"3.6"` as a dependency since the package sin't explicitly.

Reply:
`0.13.0` does not have `mock` as a dependency.

It was my understanding that we anyway were building for 3.8-3.11 thanks to the constraints in CBC, and 3.6 was therefore excluded

Reply:
>It was my understanding that we anyway were building for 3.8-3.11 thanks to the constraints in CBC, and 3.6 was therefore excluded

Indeed. But I like to think that our packages should be reproducible. Note the word think and should. A lot of our packages are not reproducible, but we should at least work towards that goal anyway.

If the package doesn't specify which Python version it's compatible with, then the requirements should reflect that.